tests/test_bundle.py

```diff
@@ -1011,6 +1011,28 @@ def test_enumeration_bundles(self):
             for x in res.json()['bundles']:
                 self.assertNotIn(x, page_one)
 
+    def test_enumeration_bundles_prefix(self):
+        bundle_uuid, bundle_version = self._put_bundle()
+
+        with self.subTest('Test successful case finds full uuid prefix.'):
+            res = self.app.get(f"/v1/bundles/all", params=dict(replica="aws", per_page=10, prefix=bundle_uuid))
+            self.assertEquals(res.json()['bundles'][0]['uuid'], bundle_uuid)
+            self.assertIn(res.json()['bundles'][0]['uuid'], res.json()['search_prefix'])
+            self.assertEquals(res.status_code, requests.codes.okay)
+
+        with self.subTest('Test successful case finds partial uuid prefix.'):
+            partial_uuid = bundle_uuid[:8]
+            res = self.app.get(f"/v1/bundles/all", params=dict(replica="aws", per_page=10, prefix=partial_uuid))
```

Comment:
This should test both uppercase prefix and lowercase prefix scenarios

Reply:
Will add.

Reply:
Done.